src/python_testing/TC_MWOCTRL_2_2.py

```diff
@@ -89,6 +90,13 @@ async def test_TC_MWOCTRL_2_2(self):
         powerValue = await self.read_mwoctrl_attribute_expect_success(endpoint=endpoint, attribute=attributes.PowerSetting)
         asserts.assert_true(powerValue == newPowerValue, "PowerSetting was not correctly set")
 
+        self.step(5)
+        newPowerValue = 125
+        try:
+            await self.send_single_cmd(cmd=commands.SetCookingParameters(powerSetting=newPowerValue), endpoint=endpoint)
+        except InteractionModelError as e:
```

Comment:
if you expect an exception, you should add an assert failure after the await to say "constraint error expected but got none". Both here and below.

Otherwise document that this constraint error is optional.